src/utils/components/NodeSelectorModal/components/LabelRow.tsx

```diff
@@ -13,9 +13,19 @@ type LabelRowProps = {
   onDelete: (id: any) => void;
 };
 
-const LabelRow: React.FC<LabelRowProps> = ({ label, onChange, onDelete }) => {
+const LabelRow: FC<LabelRowProps> = ({ label, onChange, onDelete }) => {
   const { t } = useKubevirtTranslation();
   const { id, key, value } = label;
+
+  const handlePasteLabelKey = (event): void => {
+    event.preventDefault();
+    const text = event.clipboardData.getData('text');
```

Comment:
this also on firefox working? safari?

Reply:
It works on Chrome and Firefox. I can't test on Safari.